### ISSUE TITLE
Error when installing openjdk8

### ISSUE THREAD

**Issue by ali-l** [AdoptOpenJDK/homebrew-openjdk#51]
Error: Failure while executing; `/usr/bin/sudo -E -- /bin/mv -f -- /usr/local/Caskroom/adoptopenjdk8/8,192:b12/jdk8u192-b12 /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk` exited with 1. Here's the output:
mv: rename /usr/local/Caskroom/adoptopenjdk8/8,192:b12/jdk8u192-b12 to /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk: No such file or directory

**Comment by gdams** [comment]
I'm not 100% sure on this one. AFAIK all macs **should** have `/Library/Java/JavaVirtualMachines/` created by default. Did you have Xcode command line tools installed?

**Comment by ali-l** [comment]
Yes, I did have command line tools installed. I uninstalled the jdk that I previously had. I was using the old `adoptopenjdk-openjdk8` formula. Maybe the directory was removed by the uninstall process?

It seems like no one else is having this issue, so perhaps we can simply close it. Or we can add a `mkdir -p` step before the `mv` just to be safe.

Either way, thanks for responding, and for all your work on this formula!

**Comment by fmanno** [comment]
Just had the same error when trying to install openjdk9. I think the issue might be that I've just uninstalled the latest java and that deleted the virtualmachines folder

`22:51 $ brew cask uninstall java
==> Uninstalling Cask java
==> Backing Generic Artifact 'openjdk-13.jdk' up to '/usr/local/Caskroom/java/13,33:5b8a42f3905b406298b72d750b6919f6/jdk-13.jdk'.
Password:
==> Removing Generic Artifact '/Library/Java/JavaVirtualMachines/openjdk-13.jdk'.
==> Removing directories if empty:
/Library/Java/JavaVirtualMachines
==> Purging files for version 13,33:5b8a42f3905b406298b72d750b6919f6 of Cask java`

**Comment by atomboulian** [comment]
Running into the same issue - I had to wipe my previous versions of Java and start again. So I no longer have the java 12 in the folder. I do see the new jdk though even though it errored.

```anatomboulian@ip-192-168-0-162 $ ll /Library/Java/JavaVirtualMachines
total 0
drwxr-xr-x  3 root  wheel   96 Dec 12 10:45 .
drwxr-xr-x  4 root  wheel  128 Dec 12 10:45 ..
drwxr-xr-x  3 root  wheel   96 Dec 12 10:45 adoptopenjdk-8.jdk
```

Here was my output from my installation: https://github.com/AdoptOpenJDK/homebrew-openjdk/issues/266#issuecomment-565113655

**Comment by atomboulian** [comment]
Also I had done the following before this to have a fresh whack at installing Java:

```
[10:42:57] (/Library/Java/JavaVirtualMachines)
anatomboulian@ip-192-168-0-162 $ sudo rm -rf jdk1.8.0_231.jdk jdk-12.0.2.jdk
```
So my JavaVirtualMachines folder is intact but no jdk exists. Not sure how to work around this now

**Comment by atomboulian** [comment]
Ok so I reinstalled Java 13 and it's my default, then ran the steps into install via brew and it worked. 👍

**Comment by philsherry** [comment]
I've had this a few times now, but it's the first time I've found this Issue.

> Error: adoptopenjdk8: Failure while executing; `/usr/bin/sudo -E -- /bin/rmdir -- /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/sample/try-with-resources/src` exited with .
